cypress/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "noEmit": true,
+    "target": "es5",
+    "lib": ["es5", "dom"],
```

Comment:
This means it compiles ES6 to ES5?

Reply:
My understanding is that with a es5 target es6+ will get transpiled to es5 equivs